xbmc/platform/android/storage/AndroidStorageProvider.cpp

```diff
@@ -414,3 +416,70 @@ bool CAndroidStorageProvider::PumpDriveChangeEvents(IStorageEventsCallback *call
   m_removableDrives = std::move(drives);
   return changed;
 }
+
+bool CAndroidStorageProvider::GetStorageUsage(const std::string& path, std::string& usage)
+{
+#define GIGABYTES 1073741824
```

Comment:
maybe move this to constexpr in an anonymous namespace within this file?

Reply:
Right, I just moved the function, update the code.